Index: project/app/src/main/java/com/example/geosnap/databases/PicturesDB.java
===================================================================
diff --git a/project/app/src/main/java/com/example/geosnap/databases/PicturesDB.java b/project/app/src/main/java/com/example/geosnap/databases/PicturesDB.java
deleted file mode 100644
--- a/project/app/src/main/java/com/example/geosnap/databases/PicturesDB.java	
+++ /dev/null	
@@ -1,4 +0,0 @@
-package com.example.geosnap.databases;
-
-public class PicturesDB {
-}
